src/ergw_app.erl

```diff
@@ -22,10 +22,10 @@
 start(_StartType, _StartArgs) ->
     do([error_m ||
 	   gtp_config:init(),
-	   ergw_prometheus:declare(),
 	   ensure_jobs_queues(),
 	   Pid <- ergw_sup:start_link(),
 	   ergw_config:load_config(setup:get_all_env(ergw)),
+	   ergw_prometheus:declare(),
```

Comment:
there is a small chance that moving it here is not safe. Needs further checks.

Reply:
what if we move `prometheus_histogram:declare([{name, gtp_path_rtt_milliseconds ...` into other exported function and will call it after option validation?

Reply:
problem is that ergw_config:load_config starts the sockets. The moment the socket is up, it could try to change a metric that might not declared at that point.

Reply:
this changes will reverted after merge https://github.com/travelping/ergw/pull/310

Reply:
Done. Thanks.